24-10-11.10:19:56.311 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 23780 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.10:19:56.308 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.10:19:56.316 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.10:19:58.289 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.10:19:58.293 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.10:19:58.370 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 49 ms. Found 0 Redis repository interfaces.
24-10-11.10:19:58.590 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.10:20:00.007 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.10:20:00.034 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.10:20:00.035 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.10:20:00.036 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.10:20:00.633 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.10:20:00.634 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 4185 ms
24-10-11.10:20:01.385 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.10:20:01.411 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.10:20:03.237 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.10:20:04.868 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.10:20:07.151 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.10:20:08.061 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.10:20:08.092 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.10:20:08.103 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.10:20:08.122 [main            ] INFO  Application            - Started Application in 12.88 seconds (JVM running for 15.605)
24-10-11.10:20:08.259 [RMI TCP Connection(5)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.10:20:08.259 [RMI TCP Connection(5)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.10:20:08.261 [RMI TCP Connection(5)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.10:20:08.697 [RMI TCP Connection(7)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.10:20:11.742 [RMI TCP Connection(7)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.10:46:40.233 [http-nio-8080-exec-2] INFO  Cookie                 - A cookie header was received [Hm_lvt_59b1c03e89ca78358b9dcd9d7327a300=1714459265,1714460039;] that contained an invalid cookie. That cookie will be ignored.
 Note: further occurrences of this error will be logged at DEBUG level.
24-10-11.10:51:28.446 [http-nio-8080-exec-8] ERROR [dispatcherServlet]    - Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is com.sdwu.x.api.types.exception.XApiException{code='0001', info='null'}] with root cause
com.sdwu.types.exception.AppException: null
	at com.sdwu.trigger.http.LoginController.login(LoginController.java:33)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:205)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:150)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:117)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1072)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:965)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:555)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:623)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:209)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153)
	at cn.dev33.satoken.filter.SaPathCheckFilterForServlet.doFilter(SaPathCheckFilterForServlet.java:55)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153)
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:96)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:481)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:130)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:390)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:926)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1791)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.10:52:00.614 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.10:52:00.625 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.10:52:08.940 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.10:52:08.943 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 16580 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.10:52:08.944 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.10:52:09.796 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.10:52:09.797 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.10:52:09.825 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 17 ms. Found 0 Redis repository interfaces.
24-10-11.10:52:09.895 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.10:52:10.328 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.10:52:10.337 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.10:52:10.337 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.10:52:10.337 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.10:52:10.562 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.10:52:10.567 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1576 ms
24-10-11.10:52:11.006 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.10:52:11.017 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.10:52:11.785 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.10:52:12.197 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.10:52:12.419 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.10:52:13.122 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.10:52:13.152 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.10:52:13.161 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.10:52:13.174 [main            ] INFO  Application            - Started Application in 4.702 seconds (JVM running for 6.038)
24-10-11.10:52:13.831 [RMI TCP Connection(3)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.10:52:13.831 [RMI TCP Connection(3)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.10:52:13.833 [RMI TCP Connection(3)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.10:52:13.834 [RMI TCP Connection(4)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.10:52:14.262 [RMI TCP Connection(4)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.10:52:44.071 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.10:52:44.084 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.10:52:50.967 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 12740 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.10:52:50.967 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.10:52:50.968 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.10:52:51.633 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.10:52:51.634 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.10:52:51.656 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 14 ms. Found 0 Redis repository interfaces.
24-10-11.10:52:51.715 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.10:52:52.058 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.10:52:52.065 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.10:52:52.065 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.10:52:52.065 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.10:52:52.203 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.10:52:52.203 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1200 ms
24-10-11.10:52:52.443 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.10:52:52.451 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.10:52:53.019 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.10:52:53.469 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.10:52:53.674 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.10:52:54.279 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.10:52:54.314 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.10:52:54.324 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.10:52:54.336 [main            ] INFO  Application            - Started Application in 3.727 seconds (JVM running for 4.564)
24-10-11.10:52:54.748 [RMI TCP Connection(1)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.10:52:54.749 [RMI TCP Connection(1)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.10:52:54.749 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.10:52:54.750 [RMI TCP Connection(1)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 1 ms
24-10-11.10:52:55.401 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.11:00:46.261 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.11:00:46.272 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.11:00:53.371 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.11:00:53.371 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 14608 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.11:00:53.372 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.11:00:54.077 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.11:00:54.078 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.11:00:54.102 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 15 ms. Found 0 Redis repository interfaces.
24-10-11.11:00:54.164 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.11:00:54.572 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.11:00:54.580 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.11:00:54.581 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.11:00:54.581 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.11:00:54.737 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.11:00:54.737 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1334 ms
24-10-11.11:00:54.987 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.11:00:54.995 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.11:00:55.728 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.11:00:56.353 [redisson-netty-1-7] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.11:00:56.403 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.11:00:57.261 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.11:00:57.292 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.11:00:57.302 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.11:00:57.317 [main            ] INFO  Application            - Started Application in 4.337 seconds (JVM running for 5.222)
24-10-11.11:00:57.834 [RMI TCP Connection(2)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.11:00:57.835 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.11:00:57.836 [RMI TCP Connection(3)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.11:00:57.837 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.11:00:58.334 [RMI TCP Connection(3)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.11:02:36.380 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.11:02:36.390 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.11:02:43.485 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.11:02:43.485 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 3212 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.11:02:43.486 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.11:02:44.145 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.11:02:44.147 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.11:02:44.169 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 14 ms. Found 0 Redis repository interfaces.
24-10-11.11:02:44.227 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.11:02:44.563 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.11:02:44.569 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.11:02:44.569 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.11:02:44.569 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.11:02:44.702 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.11:02:44.703 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1185 ms
24-10-11.11:02:44.938 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.11:02:44.945 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.11:02:45.504 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.11:02:45.889 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.11:02:46.303 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.11:02:46.932 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.11:02:46.966 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.11:02:46.976 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.11:02:46.988 [main            ] INFO  Application            - Started Application in 3.864 seconds (JVM running for 4.677)
24-10-11.11:02:47.296 [RMI TCP Connection(2)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.11:02:47.296 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.11:02:47.297 [RMI TCP Connection(1)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.11:02:47.298 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.11:02:47.917 [RMI TCP Connection(1)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.11:47:17.420 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.11:47:17.432 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.13:16:22.770 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.13:16:22.771 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 25380 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.13:16:22.773 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.13:16:23.473 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.13:16:23.474 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.13:16:23.511 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 19 ms. Found 0 Redis repository interfaces.
24-10-11.13:16:23.573 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.13:16:23.939 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.13:16:23.948 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.13:16:23.948 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.13:16:23.949 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.13:16:24.096 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.13:16:24.096 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1288 ms
24-10-11.13:16:24.334 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.13:16:24.343 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.13:16:24.944 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.13:16:25.256 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.13:16:25.314 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.13:16:25.947 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.13:16:25.977 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.13:16:25.986 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.13:16:25.998 [main            ] INFO  Application            - Started Application in 3.605 seconds (JVM running for 4.516)
24-10-11.13:16:26.688 [RMI TCP Connection(1)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.13:16:26.688 [RMI TCP Connection(1)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.13:16:26.690 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.13:16:26.690 [RMI TCP Connection(1)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.13:16:27.024 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.13:21:11.104 [http-nio-8080-exec-8] INFO  Cookie                 - A cookie header was received [Hm_lvt_b393d153aeb26b46e9431fabaf0f6190=1704720290,1706270606,1706270791,1706283485;] that contained an invalid cookie. That cookie will be ignored.
 Note: further occurrences of this error will be logged at DEBUG level.
24-10-11.13:21:11.107 [http-nio-8080-exec-8] WARN  DefaultHandlerExceptionResolver - Resolved [org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'GET' not supported]
24-10-11.13:29:33.534 [HikariPool-1 housekeeper] WARN  HikariPool             - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=3m36s461ms634µs100ns).
24-10-11.13:56:36.794 [redisson-netty-1-3] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x748a924c, L:/10.40.75.142:7701 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.13:56:36.795 [redisson-netty-1-8] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xb5db0d8b, L:/10.40.75.142:7704 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.13:56:36.795 [redisson-netty-1-2] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xefab4e9c, L:/10.40.75.142:7700 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.13:56:36.794 [redisson-netty-1-5] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x68b5aadc, L:/10.40.75.142:7702 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.13:56:36.794 [redisson-netty-1-10] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xfd260744, L:/10.40.75.142:7705 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.13:56:36.794 [redisson-netty-1-12] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xa0f3bec5, L:/10.40.75.142:7706 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.14:00:03.428 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@38418974 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.437 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@607a80fb (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.441 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6557a7d7 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.444 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@51e98820 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.449 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5e4172c6 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.453 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6a085cad (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.460 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@32b92724 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.466 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@52cb5776 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.471 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2930da0c (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:03.481 [http-nio-8080-exec-5] WARN  PoolBase               - HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@667d1438 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.
24-10-11.14:00:25.482 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.14:00:47.738 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.14:00:52.067 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.14:00:52.068 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 16976 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.14:00:52.069 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.14:00:52.875 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.14:00:52.876 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.14:00:52.907 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 18 ms. Found 0 Redis repository interfaces.
24-10-11.14:00:53.001 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.14:00:53.412 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.14:00:53.421 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.14:00:53.422 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.14:00:53.422 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.14:00:53.634 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.14:00:53.635 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1525 ms
24-10-11.14:00:53.938 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.14:00:53.948 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.14:00:54.913 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.14:00:55.301 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.14:00:55.356 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.14:00:56.150 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.14:00:56.178 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.14:00:56.187 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.14:00:56.201 [main            ] INFO  Application            - Started Application in 4.593 seconds (JVM running for 5.952)
24-10-11.14:00:56.779 [RMI TCP Connection(3)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.14:00:56.779 [RMI TCP Connection(3)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.14:00:56.780 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.14:00:56.782 [RMI TCP Connection(3)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.14:00:57.308 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.14:01:01.623 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.14:01:01.634 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.14:01:04.654 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.14:01:04.654 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 15184 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.14:01:04.656 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.14:01:05.286 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.14:01:05.287 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.14:01:05.310 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 14 ms. Found 0 Redis repository interfaces.
24-10-11.14:01:05.376 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.14:01:05.711 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.14:01:05.718 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.14:01:05.718 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.14:01:05.718 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.14:01:05.847 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.14:01:05.847 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1162 ms
24-10-11.14:01:06.057 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.14:01:06.065 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.14:01:06.629 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.14:01:06.923 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.14:01:06.956 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.14:01:07.540 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.14:01:07.570 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.14:01:07.580 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.14:01:07.594 [main            ] INFO  Application            - Started Application in 3.357 seconds (JVM running for 4.269)
24-10-11.14:01:07.988 [RMI TCP Connection(3)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.14:01:07.989 [RMI TCP Connection(3)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.14:01:07.991 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.14:01:07.991 [RMI TCP Connection(3)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.14:01:08.395 [RMI TCP Connection(2)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.14:01:11.643 [http-nio-8080-exec-1] INFO  Cookie                 - A cookie header was received [Hm_lvt_b393d153aeb26b46e9431fabaf0f6190=1704720290,1706270606,1706270791,1706283485;] that contained an invalid cookie. That cookie will be ignored.
 Note: further occurrences of this error will be logged at DEBUG level.
24-10-11.14:02:47.721 [redisson-netty-1-2] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xfebcc684, L:/10.40.75.142:14709 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.14:02:47.721 [redisson-netty-1-3] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x60bbc000, L:/10.40.75.142:14710 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.14:02:47.721 [redisson-netty-1-5] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x82c783bc, L:/10.40.75.142:14712 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.14:02:47.721 [redisson-netty-1-12] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x4d1fa5ec, L:/10.40.75.142:14715 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.14:02:47.721 [redisson-netty-1-10] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x2adcdff7, L:/10.40.75.142:14714 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.14:02:47.721 [redisson-netty-1-8] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xf9652c29, L:/10.40.75.142:14713 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.14:05:18.845 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.14:05:18.862 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.14:05:22.277 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.14:05:22.277 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 21408 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.14:05:22.279 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.14:05:23.020 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.14:05:23.021 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.14:05:23.045 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 15 ms. Found 0 Redis repository interfaces.
24-10-11.14:05:23.124 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.14:05:23.499 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.14:05:23.506 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.14:05:23.507 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.14:05:23.507 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.14:05:23.667 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.14:05:23.667 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1352 ms
24-10-11.14:05:23.910 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.14:05:23.917 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.14:05:24.512 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.14:05:24.922 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.14:05:25.041 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.14:05:25.711 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.14:05:25.741 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.14:05:25.750 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.14:05:25.763 [main            ] INFO  Application            - Started Application in 3.918 seconds (JVM running for 4.969)
24-10-11.14:05:26.349 [RMI TCP Connection(2)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.14:05:26.349 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.14:05:26.351 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.14:05:26.351 [RMI TCP Connection(1)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.14:05:26.631 [RMI TCP Connection(1)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.14:05:57.152 [http-nio-8080-exec-1] INFO  Cookie                 - A cookie header was received [Hm_lvt_b393d153aeb26b46e9431fabaf0f6190=1704720290,1706270606,1706270791,1706283485;] that contained an invalid cookie. That cookie will be ignored.
 Note: further occurrences of this error will be logged at DEBUG level.
24-10-11.15:03:50.055 [HikariPool-1 housekeeper] WARN  HikariPool             - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=2m22s675ms223µs600ns).
24-10-11.15:51:07.356 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.15:51:07.370 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.16:16:08.029 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.16:16:08.033 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 2008 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.16:16:08.034 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.16:16:09.044 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.16:16:09.046 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.16:16:09.085 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 21 ms. Found 0 Redis repository interfaces.
24-10-11.16:16:09.195 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.16:16:09.668 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.16:16:09.678 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.16:16:09.679 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.16:16:09.679 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.16:16:09.887 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.16:16:09.888 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 1802 ms
24-10-11.16:16:10.184 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.16:16:10.194 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.16:16:10.984 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.16:16:23.305 [redisson-netty-1-12] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.16:16:23.483 [redisson-netty-1-21] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.16:16:24.601 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.16:16:24.629 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.16:16:24.642 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.16:16:24.657 [main            ] INFO  Application            - Started Application in 17.188 seconds (JVM running for 18.628)
24-10-11.16:16:33.809 [http-nio-8080-exec-1] INFO  Cookie                 - A cookie header was received [Hm_lvt_59b1c03e89ca78358b9dcd9d7327a300=1714459265,1714460039;] that contained an invalid cookie. That cookie will be ignored.
 Note: further occurrences of this error will be logged at DEBUG level.
24-10-11.16:16:33.816 [http-nio-8080-exec-1] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.16:16:33.817 [http-nio-8080-exec-1] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.16:16:33.819 [http-nio-8080-exec-1] INFO  DispatcherServlet      - Completed initialization in 2 ms
24-10-11.16:16:44.820 [RMI TCP Connection(6)-172.20.10.3] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.16:17:53.058 [RMI TCP Connection(6)-172.20.10.3] ERROR HikariPool             - HikariPool-1 - Exception during pool initialization.
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at org.springframework.jdbc.datasource.DataSourceUtils.fetchConnection(DataSourceUtils.java:159)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:117)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:330)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.getProduct(DataSourceHealthIndicator.java:122)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.doDataSourceHealthCheck(DataSourceHealthIndicator.java:105)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.doHealthCheck(DataSourceHealthIndicator.java:100)
	at org.springframework.boot.actuate.health.AbstractHealthIndicator.health(AbstractHealthIndicator.java:82)
	at org.springframework.boot.actuate.health.HealthIndicator.getHealth(HealthIndicator.java:37)
	at org.springframework.boot.actuate.health.HealthEndpoint.getHealth(HealthEndpoint.java:94)
	at org.springframework.boot.actuate.health.HealthEndpoint.getHealth(HealthEndpoint.java:41)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getLoggedHealth(HealthEndpointSupport.java:172)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getContribution(HealthEndpointSupport.java:145)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getAggregateContribution(HealthEndpointSupport.java:156)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getContribution(HealthEndpointSupport.java:141)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getHealth(HealthEndpointSupport.java:110)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getHealth(HealthEndpointSupport.java:81)
	at org.springframework.boot.actuate.health.HealthEndpoint.health(HealthEndpoint.java:88)
	at org.springframework.boot.actuate.health.HealthEndpoint.health(HealthEndpoint.java:78)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:282)
	at org.springframework.boot.actuate.endpoint.invoke.reflect.ReflectiveOperationInvoker.invoke(ReflectiveOperationInvoker.java:74)
	at org.springframework.boot.actuate.endpoint.annotation.AbstractDiscoveredOperation.invoke(AbstractDiscoveredOperation.java:60)
	at org.springframework.boot.actuate.endpoint.jmx.EndpointMBean.invoke(EndpointMBean.java:124)
	at org.springframework.boot.actuate.endpoint.jmx.EndpointMBean.invoke(EndpointMBean.java:97)
	at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:809)
	at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl.doOperation(RMIConnectionImpl.java:1466)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl$PrivilegedOperation.run(RMIConnectionImpl.java:1307)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl.doPrivilegedOperation(RMIConnectionImpl.java:1399)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:827)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor19.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.mysql.cj.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:167)
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:89)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:144)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 61 common frames omitted
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.base/java.net.PlainSocketImpl.waitForConnect(Native Method)
	at java.base/java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:107)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:591)
	at com.mysql.cj.protocol.StandardSocketFactory.connect(StandardSocketFactory.java:155)
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:63)
	... 63 common frames omitted
24-10-11.16:17:53.061 [RMI TCP Connection(6)-172.20.10.3] WARN  DataSourceHealthIndicator - DataSource health check failed
org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:83)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:330)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.getProduct(DataSourceHealthIndicator.java:122)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.doDataSourceHealthCheck(DataSourceHealthIndicator.java:105)
	at org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator.doHealthCheck(DataSourceHealthIndicator.java:100)
	at org.springframework.boot.actuate.health.AbstractHealthIndicator.health(AbstractHealthIndicator.java:82)
	at org.springframework.boot.actuate.health.HealthIndicator.getHealth(HealthIndicator.java:37)
	at org.springframework.boot.actuate.health.HealthEndpoint.getHealth(HealthEndpoint.java:94)
	at org.springframework.boot.actuate.health.HealthEndpoint.getHealth(HealthEndpoint.java:41)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getLoggedHealth(HealthEndpointSupport.java:172)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getContribution(HealthEndpointSupport.java:145)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getAggregateContribution(HealthEndpointSupport.java:156)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getContribution(HealthEndpointSupport.java:141)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getHealth(HealthEndpointSupport.java:110)
	at org.springframework.boot.actuate.health.HealthEndpointSupport.getHealth(HealthEndpointSupport.java:81)
	at org.springframework.boot.actuate.health.HealthEndpoint.health(HealthEndpoint.java:88)
	at org.springframework.boot.actuate.health.HealthEndpoint.health(HealthEndpoint.java:78)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:282)
	at org.springframework.boot.actuate.endpoint.invoke.reflect.ReflectiveOperationInvoker.invoke(ReflectiveOperationInvoker.java:74)
	at org.springframework.boot.actuate.endpoint.annotation.AbstractDiscoveredOperation.invoke(AbstractDiscoveredOperation.java:60)
	at org.springframework.boot.actuate.endpoint.jmx.EndpointMBean.invoke(EndpointMBean.java:124)
	at org.springframework.boot.actuate.endpoint.jmx.EndpointMBean.invoke(EndpointMBean.java:97)
	at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:809)
	at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl.doOperation(RMIConnectionImpl.java:1466)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl$PrivilegedOperation.run(RMIConnectionImpl.java:1307)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl.doPrivilegedOperation(RMIConnectionImpl.java:1399)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:827)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor19.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:906)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:831)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at org.springframework.jdbc.datasource.DataSourceUtils.fetchConnection(DataSourceUtils.java:159)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:117)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	... 47 common frames omitted
Caused by: com.mysql.cj.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:167)
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:89)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:144)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:850)
	... 61 common frames omitted
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.base/java.net.PlainSocketImpl.waitForConnect(Native Method)
	at java.base/java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:107)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:591)
	at com.mysql.cj.protocol.StandardSocketFactory.connect(StandardSocketFactory.java:155)
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:63)
	... 63 common frames omitted
24-10-11.16:17:53.064 [RMI TCP Connection(6)-172.20.10.3] WARN  HealthEndpointSupport  - Health contributor org.springframework.boot.actuate.jdbc.DataSourceHealthIndicator (db) took 68245ms to respond
24-10-11.16:26:35.056 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.16:26:35.061 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 3016 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.16:26:35.063 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.16:26:36.252 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.16:26:36.254 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.16:26:36.287 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 22 ms. Found 0 Redis repository interfaces.
24-10-11.16:26:36.403 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.16:26:37.068 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.16:26:37.081 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.16:26:37.082 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.16:26:37.082 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.16:26:37.361 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.16:26:37.361 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 2228 ms
24-10-11.16:26:37.871 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.16:26:37.902 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.16:26:39.019 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.16:26:39.632 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.16:26:39.856 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.16:26:40.786 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.16:26:40.823 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.16:26:40.835 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.16:26:40.854 [main            ] INFO  Application            - Started Application in 6.509 seconds (JVM running for 8.643)
24-10-11.16:26:41.713 [RMI TCP Connection(1)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.16:26:41.713 [RMI TCP Connection(1)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.16:26:41.716 [RMI TCP Connection(3)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.16:26:41.719 [RMI TCP Connection(1)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 5 ms
24-10-11.16:26:42.218 [RMI TCP Connection(3)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.18:10:58.519 [redisson-netty-1-2] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x220fdab2, L:/10.40.75.142:7060 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.18:10:58.519 [redisson-netty-1-10] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xb56ff9de, L:/10.40.75.142:7064 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.18:10:58.688 [redisson-netty-1-5] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x8fa544aa, L:/10.40.75.142:7062 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.18:10:58.689 [redisson-netty-1-3] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x8b0254ed, L:/10.40.75.142:7061 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.18:10:58.717 [redisson-netty-1-8] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0xf0199d37, L:/10.40.75.142:7063 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.18:10:59.731 [redisson-netty-1-12] ERROR ErrorsLoggingHandler   - Exception occured. Channel: [id: 0x64e33031, L:/10.40.75.142:7065 - R:10.34.34.73/10.34.34.73:6379]
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:256)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
24-10-11.18:11:01.675 [HikariPool-1 housekeeper] WARN  HikariPool             - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=1h21m19s266ms178µs400ns).
24-10-11.18:12:36.700 [http-nio-8080-exec-6] INFO  Cookie                 - A cookie header was received [Hm_lvt_59b1c03e89ca78358b9dcd9d7327a300=1714459265,1714460039;] that contained an invalid cookie. That cookie will be ignored.
 Note: further occurrences of this error will be logged at DEBUG level.
24-10-11.18:38:31.929 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.18:38:31.958 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
24-10-11.18:38:46.281 [main            ] INFO  Application            - Starting Application using Java 11.0.1 on LAPTOP-JSOKR72D with PID 3772 (D:\qiniu-collector\collector\collector-app\target\classes started by 13928 in D:\qiniu-collector\collector)
24-10-11.18:38:46.284 [background-preinit] INFO  Version                - HV000001: Hibernate Validator 6.2.5.Final
24-10-11.18:38:46.285 [main            ] INFO  Application            - The following 1 profile is active: "dev"
24-10-11.18:38:47.540 [main            ] INFO  RepositoryConfigurationDelegate - Multiple Spring Data modules found, entering strict repository configuration mode
24-10-11.18:38:47.542 [main            ] INFO  RepositoryConfigurationDelegate - Bootstrapping Spring Data Redis repositories in DEFAULT mode.
24-10-11.18:38:47.578 [main            ] INFO  RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 23 ms. Found 0 Redis repository interfaces.
24-10-11.18:38:47.698 [main            ] WARN  ClassPathMapperScanner - No MyBatis mapper was found in '[com.sdwu.config]' package. Please check your configuration.
24-10-11.18:38:48.215 [main            ] INFO  TomcatWebServer        - Tomcat initialized with port(s): 8080 (http)
24-10-11.18:38:48.225 [main            ] INFO  Http11NioProtocol      - Initializing ProtocolHandler ["http-nio-8080"]
24-10-11.18:38:48.226 [main            ] INFO  StandardService        - Starting service [Tomcat]
24-10-11.18:38:48.226 [main            ] INFO  StandardEngine         - Starting Servlet engine: [Apache Tomcat/9.0.75]
24-10-11.18:38:48.433 [main            ] INFO  [/]                    - Initializing Spring embedded WebApplicationContext
24-10-11.18:38:48.434 [main            ] INFO  ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 2088 ms
24-10-11.18:38:48.777 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join properties config complete
24-10-11.18:38:48.788 [main            ] INFO  MybatisPlusJoinAutoConfiguration - mybatis plus join SqlInjector init
24-10-11.18:38:49.849 [main            ] INFO  Version                - Redisson 3.26.0
24-10-11.18:38:50.841 [redisson-netty-1-6] INFO  ConnectionsHolder      - 1 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.18:38:51.004 [redisson-netty-1-13] INFO  ConnectionsHolder      - 5 connections initialized for 10.34.34.73/10.34.34.73:6379
24-10-11.18:38:52.112 [main            ] INFO  EndpointLinksResolver  - Exposing 1 endpoint(s) beneath base path '/actuator'
24-10-11.18:38:52.157 [main            ] INFO  Http11NioProtocol      - Starting ProtocolHandler ["http-nio-8080"]
24-10-11.18:38:52.170 [main            ] INFO  TomcatWebServer        - Tomcat started on port(s): 8080 (http) with context path ''
24-10-11.18:38:52.193 [main            ] INFO  Application            - Started Application in 6.507 seconds (JVM running for 8.455)
24-10-11.18:38:53.050 [RMI TCP Connection(2)-10.40.75.142] INFO  [/]                    - Initializing Spring DispatcherServlet 'dispatcherServlet'
24-10-11.18:38:53.051 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Initializing Servlet 'dispatcherServlet'
24-10-11.18:38:53.055 [RMI TCP Connection(2)-10.40.75.142] INFO  DispatcherServlet      - Completed initialization in 4 ms
24-10-11.18:38:53.057 [RMI TCP Connection(1)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Starting...
24-10-11.18:38:53.625 [RMI TCP Connection(1)-10.40.75.142] INFO  HikariDataSource       - HikariPool-1 - Start completed.
24-10-11.18:48:32.093 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown initiated...
24-10-11.18:48:32.107 [SpringApplicationShutdownHook] INFO  HikariDataSource       - HikariPool-1 - Shutdown completed.
